deploy/crds/enterprise.splunk.com_clustermasters_crd.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   scope: Namespaced
   versions:
   - additionalPrinterColumns:
-    - description: Status of cluster master
+    - description: Status of cluster manager
```

Comment:
Aren't these changes supposed to be independent of the release changes? Not sure if I am missing something. Are we making the changes from master to manager explicitly here along with the release changes?

Reply:
Seems like this is a missing change on Develop. I can see the same on my local build.

Reply:
This wasn't intended. I believe is caused because of this [change](https://github.com/splunk/splunk-operator/pull/497/files#diff-2c68c73ef5efbbac3a76e3eb43c32c228331cfcab095dbf808d0810942a40196). 

It shouldn't be harmful but we might want to roll back to keep it consistent for the release.  @pdhanoyasplunk , @smohan-splunk.  If we roll back we'll have to rerun the workflow. What do you think?

Reply:
I agree, although not harmful, its better we roll back since CRD version doesn't change this release. We can bring these back with MC CRD changes post 1.0.3 release

Reply:
After further discussion, we decided to keep the changes